gallery/visualtests/nxRadio.js

```diff
@@ -85,4 +86,9 @@ describe('NxRadio', function() {
     it('looks disabled by default', simpleTest(disabledSelector));
     it('looks disabled when hovered', hoverTest(disabledSelector));
   });
+
+  describe('Attribute-Disabled-Checked NxRadio', function() {
+    it('looks disabled by default', simpleTest(disabledCheckedSelector));
+    it('looks disabled when hovered', hoverTest(disabledCheckedSelector));
```

Comment:
After running CI be sure the go into applitools and check that the the test results came out as expected.  This is a standard part of adding new visual tests - the test infrastructure can't know whether new screenshots are correct or not